storybook/stories/SetStateExample.js

```diff
@@ -0,0 +1,99 @@
+import React, { Component } from 'react';
+import Script from 'react-load-script';
+import GooglePlaces from '../../src/GooglePlaces';
+import { action } from '@kadira/storybook';
+
+import '../../styles/index.css';
+
+class SetStateExample extends Component {
+  state = {
+    googleApiLoaded: false,
+    googleApiError: false,
+    result: null,
+  };
+
+  onGoogleApiLoaded = () => {
+    action('googleApiLoaded')(true); // Log to storybook's "action-logger"
+    this.setState({ googleApiLoaded: true });
+    console.log('googleApiLoaded');
+  };
+
+  onGoogleApiError = error => {
+    action('googleApiError')(true); // Log to storybook's "action-logger"
+    this.setState({ googleApiError: true });
+    console.log('googleApiError');
+  };
+
+  onSelect = result => {
+    action('onSelect')(result); // Log to storybook's "action-logger"
+    this.setState(result);
+  };
+
+  render() {
+    const { googleApiLoaded } = this.state;
+    return (
+      <div>
+        {!googleApiLoaded
+          ? <Script
+              url="https://maps.googleapis.com/maps/api/js?key=AIzaSyBDvgbceyFAP8QbGpNSVt45PPGDUylFokI&amp;libraries=places"
```

Comment:
Should require that a user enter their own key as a prerequisite for running this, rather than just hardcoding one in.